pkg/downloader/chart_downloader.go

```diff
@@ -109,6 +111,7 @@ func (c *ChartDownloader) DownloadTo(ref, version, dest string) (string, *proven
 
 	destfile := filepath.Join(dest, name)
 	if err := fileutil.AtomicWriteFile(destfile, data, 0644); err != nil {
+		fmt.Fprintf(&out, "Chart downloaded to: %s\n", destfile)
```

Comment:
Shouldn't this use ChartLoader.Out to be consistent with the rest of the code?

```suggestion
		fmt.Fprintf(c.Out, "Chart downloaded to: %s\n", destfile)
```